<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task3_ControllerTVClass</class>
 <widget class="QMainWindow" name="Task3_ControllerTVClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task3_ControllerTV</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QPushButton" name="numb3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="numb8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="numb7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="numb2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="numb0">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="numb1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="numb5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="numb4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="numb6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="numb9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="canalPrev">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="canalNext">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>+C</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="volDown">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="volUp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>+V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="lab2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Volume: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lab1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Canal: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="canalLab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="volLab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Task3_ControllerTV.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>numb1</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>numb1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numb4</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>numb4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numb7</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>numb7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numb2</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>numb2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numb3</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>numb3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numb5</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>numb5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numb6</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>numb6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numb8</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>numb8()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numb9</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>numb9()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numb0</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>numb0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volDown</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>volDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volUp</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>volUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>canalPrev</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>canPrev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>canalNext</sender>
   <signal>clicked()</signal>
   <receiver>Task3_ControllerTVClass</receiver>
   <slot>canNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>numb1()</slot>
  <slot>numb2()</slot>
  <slot>numb3()</slot>
  <slot>numb4()</slot>
  <slot>numb5()</slot>
  <slot>numb6()</slot>
  <slot>numb7()</slot>
  <slot>numb8()</slot>
  <slot>numb9()</slot>
  <slot>numb0()</slot>
  <slot>canPrev()</slot>
  <slot>canNext()</slot>
  <slot>volDown()</slot>
  <slot>volUp()</slot>
 </slots>
</ui>
